openreview/conference/builder.py

```diff
@@ -1533,6 +1536,62 @@ def is_release_authors(is_note_accepted):
             self.set_homepage_decisions(decision_heading_map=decision_heading_map)
         self.client.remove_members_from_group('active_venues', self.id)
 
+    def post_decisions(self, decisions_file):
+        decisions_data = list(csv.reader(StringIO(decisions_file.decode()), delimiter=","))
+        decision_notes = {
+            n.forum: n for n in self.client.get_all_notes(
+                invitation=self.get_invitation_id(self.decision_stage.name, '.*')
+            )}
+
+        paper_notes = {
+            n.forum: n for n in self.client.get_all_notes(
+                invitation=self.get_invitation_id('Blind_Submission')
+            )}
+
+        def post_decision(paper_decision):
+            if len(paper_decision) == 3:
+                paper_id, decision, comment = paper_decision
+            elif len(paper_decision) == 2:
+                paper_id, decision = paper_decision
+                comment = ''
+            else:
+                raise ValueError(
+                    "Too many values provided in the decision file. Expected values are: paper_id, decision, comment"
```

Comment:
is it possible that `len(paper_decision) == 0`? in this case should we say "Wrong values in the decision file. Expected values are: paper_id, decision, comment". 

I would also throw an OpenReviewException instead.